lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1607,14 +1607,10 @@ void addIntelFPGADecorationsForStructMember(
 bool LLVMToSPIRV::isKnownIntrinsic(Intrinsic::ID Id) {
   // Known intrinsics usually do not need translation of their declaration
   switch (Id) {
-  case Intrinsic::assume:
```

Comment:
Note: this one actually can be translated into SPIR-V if corresponding extension is enabled

Reply:
On llvm_release_100 branch?

Reply:
Oops, I've missed the fact that this is not for `master`. Then no, it can't be translated on that branch anyway